CHANGELOG.md

```diff
@@ -8,6 +8,111 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+| [PURIS KIT](https://eclipse-tractusx.github.io/docs-kits/category/puris-kit)                         |                                                                n/a                                                                 | [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/PURIS%20Kit/PURIS%20Kit%20Changelog) |
+| [SSI Credential Issuer](https://github.com/eclipse-tractusx/ssi-credential-issuer)                   | ssi-credential-issuer: [1.0.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/ssi-credential-issuer-1.0.0) |     [1.0.0](https://github.com/eclipse-tractusx/ssi-credential-issuer/releases/tag/v1.0.0)     |
+
+### Updated
+
+| Component                                                                                                                     |                                                                                             Helm Chart (s)                                                                                             |                                                                                                                                                                                          App-/KIT Version (s)                                                                                                                                                                                          |
+|-------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+| [Business Partner KIT](https://eclipse-tractusx.github.io/docs-kits/category/business-partner-kit)                            |                                                                                                  n/a                                                                                                   |                                                                                                                                                     [4.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Business%20Partner%20Kit/Changelog)                                                                                                                                                      |
```

Comment:
add new table entry BEFORE:
| [Behavioral Twin: Model and Data processing (MDP) KIT](https://eclipse-tractusx.github.io/docs-kits/category/model-based-development-and-data-processing-mdp-kit) |      n/a       | [1.0.0](https://eclipse-tractusx.github.io/docs-kits/kits/Behaviour%20Twin%20MDP%20Kit/Changelog%20Model%20Based%20Development%20and%20Data%20Processing%20Kit) |